tests/sflow/test_sflow.py

```diff
@@ -264,19 +282,63 @@ def sflowbase_config(duthosts, rand_one_dut_hostname):
     duthost.shell("config sflow polling-interval 20")
     for port in var['sflow_ports']:
         config_sflow_interfaces(
-            duthost, port, status='enable', sample_rate='512')
+            duthost, port, status='enable', sample_rate=SFLOW_RATE_DEFAULT)
     time.sleep(2)
     verify_show_sflow(duthost, status='up', collector=[
                       'collector0', 'collector1'])
     for intf in var['sflow_ports']:
-        verify_sflow_interfaces(duthost, intf, 'up', 512)
+        verify_sflow_interfaces(duthost, intf, 'up', SFLOW_RATE_DEFAULT)
+
+
+# ----------------------------------------------------------------------------------
+
+@pytest.fixture
+def tested_portchannel_members(duthost, tbinfo):
```

Comment:
Please try to avoid using function name started with `test`. I'm not sure if this can be handled by pytest appropiately.

Reply:
Hi @bingwang-ms , the function name has renamed from `tested_portchannel_members` to `selected_portchannel_members`